languages.yaml

```
@@ -79,6 +79,43 @@ Ada:
       - "#0018C9"
       - "#0C0A7C"
     chip: "#02F88C"
+
+Arduino:
+  type: programming
+  ascii: |
+    {0}                 -+++++++++++++++                 
+    {0}             ++++++++++++++++++++++++             
+    {0}          ++++++++++++++++++++++++++++++          
+    {0}       ++++++++++++++++++++++++++++++++++++       
+    {0}      ++++++++++++++++++++++++++++++++++++++      
+    {0}    ++++++++++++++++++++++++++++++++++++++++++    
+    {0}   ++++++++++++++++++++++++++++++++++++++++++++   
+    {0}  ++++++++++++++++++++++++++++++++++++++++++++++  
+    {0} +++++++++++++---+++++++++++++++++--+++++++++++++ 
+    {0}+++++++++-..      ...++++++++...      ..++++++++++
+    {0}++++++++.   ..---..   .-++-.   ..----.   .++++++++
+    {0}+++++++.  .+++++++++-. .... ..+++++++++.  .+++++++
+    {0}++++++. ..++++++++++++.    .+++++. -++++-. .++++++
+    {0}++++++. .-++-     -++++.  .++++.     -++-. .++++++
+    {0}++++++. ..++++++++++++.    .+++++. -++++-. .++++++
+    {0}+++++++.  .+++++++++-. .... .-+++++++++.  .+++++++
+    {0}++++++++..  .----..   .-++-.   ..----.  ..++++++++
+    {0}+++++++++-..      ...++++++++...      ..++++++++++
+    {0} +++++++++++++---+++++++++++++++++-++++++++++++++ 
+    {0}  ++++++++++++++++++++++++++++++++++++++++++++++  
+    {0}   ++++++++++++++++++++++++++++++++++++++++++++   
+    {0}    ++++++++++++++++++++++++++++++++++++++++++    
+    {0}      ++++++++++++++++++++++++++++++++++++++      
+    {0}       ++++++++++++++++++++++++++++++++++++       
+    {0}          ++++++++++++++++++++++++++++++          
+    {0}             ++++++++++++++++++++++++             
+    {0}                 -+++++++++++++++                 
+  colors:
+    ansi:
+      - turquoise
```

```suggestion
      - cyan
```
Turqoise isn't an ANSI color. Cyan is probably the closest.